app/javascript/front-page/components/NewsItem.vue

```diff
@@ -92,7 +94,8 @@ export default {
   },
   props: {
     newsItem: { type: Object, required: true },
-  },
+
```

Comment:
Ich glaube hier fehlt ein `},` und dafuer unten dann eins zuviel. Aktuell wird glaub ich das "methods" damit in "props" reingeholt.